streams/src/main/java/org/apache/kafka/streams/errors/internals/DefaultErrorHandlerContext.java

```diff
@@ -51,6 +53,19 @@ public DefaultErrorHandlerContext(final String topic,
         this.taskId = taskId;
     }
 
+    public DefaultErrorHandlerContext(final ProcessorContext processorContext,
```

Comment:
Should this constructor replace the other one? And we just pass in `null` for `processorContext`?

Or we use `setRecordContext()` -- but having constructor overloads is always something I think is better to avoid if we can.

Reply:
done